File: FeedleDataTier/FeedleDataTier/Network/UpdatePostRequest.cs
using System.Text.Json.Serialization;
using System.Transactions;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class UpdatePostRequest : Request
    {
        [JsonPropertyName("post")]
        public Post Post { get; set; }

        public UpdatePostRequest(Post post) : base(RequestType.UpdatePost)
        {
            this.Post = post;
        }
    }
}

File: Backend/src/main/java/com/feedle/feedleapi/Networking/UpdatePostRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.Post;

public class UpdatePostRequest extends Request{
    private Post post;
    public UpdatePostRequest(Post post)
    {
        super(RequestType.UpdatePost);
        this.post = post;
    }

    public Post getPost() {
        return post;
    }
}
